File: ecommerc_dotnet/dto/Response/OrderResponseDto.cs
namespace ecommerc_dotnet.dto.Response;

public class OrderResponseDto
{
    public Guid id { get; set; }
    public decimal longitude { get; set; }
    public decimal latitude { get; set; }
    public decimal totalPrice { get; set; }
    public String name { get; set; }
    public String userPhone { get; set; }
    public int status { get; set; }
    public List<OrderItemResponseDto> orderItems { get; set; }
}

File: ecommerce-delivery-man/app/src/main/java/com/example/e_commerc_delivery_man/dto/response/OrderResponseDto.kt
package com.example.eccomerce_app.dto.response

import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class OrderResponseDto(
    @Serializable(with = UUIDKserialize::class)
    val id: UUID,
    val longitude: Double,
    val latitude: Double,
    val userPhone: String,
    var name: String,
    val status:Int,
    val totalPrice: Double,
    val realPrice: Double,
    val deliveryFee: Double,
    val orderItems:List<OrderItemResponseDto>


)
